packages/mui-material/src/useMediaQuery/useMediaQuery.test.js

```diff
@@ -304,7 +304,7 @@ describe('useMediaQuery', () => {
   });
 
   describe('server-side', () => {
-    it('should use the ssr match media ponyfill', () => {
+    it('should use the ssr match media polyfill', () => {
```

Comment:
It's not exactly a ponyfill or a polyfill. See the discussion in #14215. I recommend removing it and changing it to:
 ```suggestion
     it('should use the SSR match media implementation', () => {
```